packages/autocomplete/src/index.ts

```diff
@@ -116,6 +121,22 @@ async function completer(params: AutocompleteParameters, line: string): Promise<
     const hits = filterShellAPI(params, SHELL_COMPLETIONS, elToComplete);
     return [hits.length ? hits : [], line];
   } else if (firstLineEl.match(/\bdb\b/) && splitLine.length === 2) {
+    if (elToComplete.match(/aggregate\s*\(\s*\[\s*\{\s*/)) {
+      const splitQuery = line.split('{');
+      const prefix = splitQuery.pop()?.trim() || '';
+      const command: string = prefix ? line.split(prefix).shift() as string : line;
+      const suggestFirstStage = splitQuery.length <= 2;
+
+      const expressions = suggestFirstStage
+        ? DB_AGGREGATE_COMPLETIONS
+        : ([] as AnyCompletions).concat(
+          BASE_COMPLETIONS,
+          getStageAccumulators(params, elToComplete)
+        );
```

Comment:
TS should also be fine with simplifying this (and this probably applies elsewhere in the file as well):

```suggestion
        : [
          ...BASE_COMPLETIONS,
          ...getStageAccumulators(params, elToComplete)
        ];
```